routes/index.js

```diff
@@ -2,8 +2,8 @@ var express = require('express');
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get('/dashboard', function(req, res, next) {
```

Comment:
I think if unauthenticated, redirect to /login. Otherwise, let the request go where it wants.